libs/perun/dialogs/src/lib/change-group-resource-assigment-dialog/change-group-resource-assigment-dialog.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="{{theme}}">
```

Comment:
please check the format of this html, add spaces where it should be, remove them where they shouldnt be